modules/reindex/src/main/java/org/elasticsearch/index/reindex/RestReindexAction.java

```diff
@@ -62,16 +63,16 @@ public String getName() {
     public RestChannelConsumer prepareRequest(RestRequest request, NodeClient client) throws IOException {
         // todo: remove system property escape hatch in 8.0
         // todo: fix version constant on backport to 7.x
-        if (clusterService.state().nodes().getMinNodeVersion().before(Version.V_8_0_0)
-                || System.getProperty("es.reindex.resilience", "true").equals("false")) {
+        if (clusterService.state().nodes().getMinNodeVersion().before(Version.V_8_0_0)) {
```

Comment:
Maybe we need a 7.x escape hatch to do the old transport action too? For those weird cases (if any) where they use lots of very small reindexes.